guides/common/modules/proc_creating-an-activation-key.adoc

```diff
@@ -1,23 +1,20 @@
 [id="Creating_an_Activation_Key_{context}"]
 = Creating an activation key
 
-You can configure the activation key to set system purpose attributes on hosts during registration.
-This can help determine which repositories are available on the host and also with reporting in the Subscriptions service of the Red Hat Hybrid Cloud Console.
-+
-. Activation key providing a custom set of subscription for auto-attach.
-+
-If there are subscriptions specified and auto-attach is enabled, hosts using the activation key select the best fitting subscription from the list specified in the activation key.
-Setting system purpose on the activation key does not affect this scenario.
-+
-. Activation key with the exact set of subscriptions.
-+
-If there are subscriptions specified and auto-attach is disabled, hosts using the activation key are associated with all subscriptions specified in the activation key.
-Setting system purpose on the activation key does not affect this scenario.
+Subscription Manager attaches subscriptions differently depending on the following factors:
 
-.{customproducttitle}s
-If a {customproduct}, typically containing content not provided by Red Hat, is assigned to an activation key, this product is always enabled for the registered content host regardless of the auto-attach setting.
+* Are there any subscriptions associated with the activation key?
+* For {RHEL} 8 hosts: Is there system purpose set on the activation key?
 
-To use the CLI instead of the {ProjectWebUI}, see the xref:cli-creating-an-activation-key_{context}[].
+Note that {Project} automatically attaches subscriptions only for the products installed on a host.
+For subscriptions that do not list products installed on {RHEL} by default, such as the Extended Update Support (EUS) subscription, use an activation key specifying the required subscriptions.
+
+Based on the previous factors, there is one possible scenario for subscribing with activation keys:
+
+* Activation key that attaches subscriptions automatically.
```

Comment:
From the beginning of this file to here, (including `* Activation key that attaches subscriptions automatically.`, should be removed.

The blurb for the procedure can be something like "Create an activation key to assign various attributes to hosts at registration time."

Reply:
@jeremylenz I don't see the text that says `Based on the previous factors, there is one possible scenario for subscribing with activation keys:` in my text editor for this .adoc file. Also `Activation key that attaches subscriptions automatically.` says `Activation key attaches subscriptions automatically.` in my text editor. Here is the text I have before `Activation key attaches subscriptions automatically`:
![image](https://github.com/theforeman/foreman-documentation/assets/91690569/4b285e73-f2f1-4812-8a3e-a6507c47a4c0)

Reply:
Should all of that stay or be removed? @jeremylenz

Reply:
All of this should be removed.

Not sure why your text editor would be showing different content than what's here. Maybe worth checking with a colleague to see if there are some git shenanigans you can do to fix it. :)